docs/topics/proc_starting-a-wildflyswarm-uberjar-in-debugging-mode.adoc

```diff
@@ -0,0 +1,29 @@
+
+[#starting-a-wildflyswarm-uberjar-in-debugging-mode]
+= Starting a {WildFlySwarm} Uberjar in Debugging Mode
+
+If you chose to package your application as a {WildFlySwarm} uberjar, debug it by executing it with the following parameters.
+
+.Prerequisites
+
+* An uberjar with your application
+
+.Procedure
+
+. In a console, navigate to the directory with the uberjar.
+. Execute the uberjar with the following parameters.
+Ensure that all the parameters are specified before the name of the uberjar on the line.
++
+--
+[source,bash,options="nowrap"]
+----
+$ java -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=$PORT_NUMBER -jar $UBERJAR_FILENAME
+----
+
+Here, `$PORT_NUMBER` is an unused port number of your choice.
+Remember this number for the remote debugger configuration.
+--
+
+.Related Information
+
+* xref:creating-an-uberjar[]
```

Comment:
This file is fairly generic -- I believe it would apply to all our Java-based runtimes, as all of Spring Boot, Vert.x and WildFly Swarm create an uberjar.